app.py

```diff
@@ -0,0 +1,24 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
+BASE_URL = 'https://api.nytimes.com/svc/search/v2/articlesearch.json'
```

Comment:
Maybe you can move the API key and BASE_URL into a .env. As  commiting the key isn't advised